src/blob/core.rs

```diff
@@ -153,6 +153,22 @@ impl Blob {
             .context("failed to create iterator for raw records")
     }
 
+    async fn dump_corrupted(&self) -> Result<()> {
+        warn!("Blob {} dump started!", self.name());
```

Comment:
Pearl users may be confused reading this message, I think we should also notice, that the blob is corrupted.